### PR TITLE
Add flask test app and configure test harness

### PR DESCRIPTION
**Pull request by RoelWKramer**
* add test env vars to pytest.ini
* fix tests to use env vars instead
* add initial test with flask test app

### CODE REVIEW

File: src/tests/test_certrequest.py [hunk 1]
```diff
@@ -10,7 +11,7 @@ def __init__(self, status_code=0, reason=""):
         self.reason = reason
 
 class TestConfig(unittest.TestCase):
-
+    """Test if requests get PKCS12 config from environment"""
     def setUp(self) -> None:
```
Comment (RoelWKramer) [inline src/tests/test_certrequest.py:15]: Not our code. but this setUp method doesnt do anything.

File: src/tests/utils/generate_jwt.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+function rs_sign() { openssl dgst -binary -sha256 -sign playback-auth-keys/private.pem ; }
+
+JWT_HDR_B64="$(echo -n "$HEADER" | b64enc)"
+JWT_PAY_B64="$(echo -n "$PAYLOAD" | b64enc)"
+UNSIGNED_JWT="$JWT_HDR_B64.$JWT_PAY_B64"
+#SIGNATURE=$(echo -n "$UNSIGNED_JWT" | rs_sign | b64enc)
+SIGNATURE=$(echo -n "$UNSIGNED_JWT" | b64enc)
+
+echo -n "$UNSIGNED_JWT.$SIGNATURE" > jwt.txt
```
Comment (RoelWKramer) [inline src/tests/utils/generate_jwt.sh:41]: new line at the end

File: src/tests/utils/generate_jwt.sh [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env bash
```
Comment (RoelWKramer) [inline src/tests/utils/generate_jwt.sh:1]: Cant we do this in python or pytest fixtures? That way we can easily pass roles to it for example.

File: src/tests/conftest.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        yield client
+
+
+@pytest.fixture
+def jwt_header() -> dict:
+    return {ACCESS_TOKEN_HEADER: _read_file(f'{BASE}/utils/jwt.txt')}
+
+
+@pytest.fixture(autouse=True)
+def stuf_310_response(requests_mock):
```
Comment (RoelWKramer) [inline src/tests/conftest.py:48]: Which test uses this? Is autouse=True set because it is used in a "traditional" TestCase-test?

Comment (roel-devries) [reply src/tests/conftest.py:48]: Just the 1 test in this pr for the moment. I thought every request needs to be mocked with this response, so we don't have to specify it in the function arguments.

File: src/tests/utils/generate_jwt.sh (hunk 3 above)
Comment (roel-devries) [reply src/tests/utils/generate_jwt.sh:1]: Let's try that idd

**Review by roel-devries — APPROVED** [review]

**Pull request merged**